### ISSUE TITLE
Error creating ALL RESULTS table and something about ForkManager

### ISSUE THREAD

**Issue by pablitoamigu** [xFFFFF/Gekko-BacktestTool#27]
Hi first of all great work, I really like this backtesttool and gekko this is better than life xD
sorry bothering you i'm trying to make this work ok, i had this error maybe you can help me.

[2018-06-03 10:03:47] Creating TOP STRATEGY table...
DBD::CSV::st execute failed: No such column 'DISTINCT strategy' at DBI/DBD/SqlEngine.pm line 1271.
 [for Statement "SELECT DISTINCT strategy, strategy_settings FROM tmp/tmp_datavz_de.csv"] at script/backtest.pl line 182.
DBD::CSV::st execute failed: No such column 'DISTINCT strategy' at DBI/DBD/SqlEngine.pm line 1271.
 [for Statement "SELECT DISTINCT strategy, strategy_settings FROM tmp/tmp_datavz_de.csv"] at script/backtest.pl line 182.
ubuntu@unnamed:~/gekkoquasar/gekko$ BTC:ETC                                       TEMA        0.00%           -2.37        0.00           %         %           %   60:10     3  00:02
The storable module was unable to store the child's data structure to the temp file "/tmp/xtngYT8gL8/Parallel-ForkManager-764-768.txt":  can't create /tmp/xtngYT8gL8/Parallel-ForkManager-764-768.txt: No such file or directory at Parallel/ForkManager.pm line 84.
BTC:ETC                                       TEMA        0.00%           -2.37        0.00           %         %           %   30:10     3  00:02
The storable module was unable to store the child's data structure to the temp file "/tmp/xtngYT8gL8/Parallel-ForkManager-764-766.txt":  can't create /tmp/xtngYT8gL8/Parallel-ForkManager-764-766.txt: No such file or directory at Parallel/ForkManager.pm line 84.
BTC:ETC                                       TEMA       -5.74%           -8.11        2.33       0.00%    -1.04%      -1.46%   15:10     3  00:02
The storable module was unable to store the child's data structure to the temp file "/tmp/xtngYT8gL8/Parallel-ForkManager-764-765.txt":  can't create /tmp/xtngYT8gL8/Parallel-ForkManager-764-765.txt: No such file or directory at Parallel/ForkManager.pm line 84.

**Comment by pkjg** [comment]
+1

**Comment by pkjg** [comment]
works fine if #strategies > warmups and pairs, else doesn't. something about forkmanager cannot start child process inside child process. I think the logic using fork manager needs to be revisited.

**Comment by pkjg** [comment]
@xFFFFF any idea about how to resolve this? It kinda restricts testing of a single strategy on multiple timeframes.

**Comment by lkru** [comment]
Did you solve this fubarred?

**Comment by fubarred** [comment]
@lkru No, couldn't. I tried but was taking too much of my time since I am not a python dev. Thought just raise the issue and keep using whatever way it works fine to gain momentum.

**Comment by lkru** [comment]
@fubarred I have it running now.
downgrade to gekko 5 or:
https://github.com/xFFFFF/Gekko-BacktestTool/issues/35
about fork manager I keep one candle size in backtest config file
